ceph_deploy/tests/unit/test_cli.py

```diff
@@ -0,0 +1,46 @@
+from ceph_deploy import cli
+from ceph_deploy.tests import util
+
+
+class FakeLogger(object):
+
+    def __init__(self):
+        self._calls = []
+        self._info = []
+
+    def _output(self):
+        return '\n'.join(self._calls)
+
+    def _record(self, level, message):
+        self._calls.append(message)
+        method = getattr(self, '_%s' % level)
+        method.append(message)
+
+    def info(self, message):
+        self._record('info', message)
+
+
+class TestLogFlags(object):
+
+    def setup(self):
+        self.logger = FakeLogger()
+
+    def test_logs_multiple_object_attributes(self):
+        args = util.Empty(verbose=True, adjust_repos=False)
+        cli.log_flags(args, logger=self.logger)
+        result = self.logger._output()
+        assert ' verbose ' in result
+        assert ' adjust_repos ' in result
```

Comment:
very good question. I felt this was OK here because the test wants to ensure that "not only one value is sent out". 

Since the signature specifies this one assert failure (even though there are two) would indicate that not all of them were present and clearly point to the failure.

This could've been a single assert but the string output was just too big to compare them. It was easier to do two.

Usually, I split them when the answer to the question is different from what the name of the test defines. For this test, I would ask: "is this assert asserting that multiple object attributes are logged?". If I had another assert, maybe something like `assert logger.info.called_once()` The answer to my question would no longer hold.